scripts/deployment/test_purposes_only_deploy_04_agent_mech.js

```diff
@@ -0,0 +1,78 @@
+/*global process*/
+
+const { ethers } = require("hardhat");
+const { LedgerSigner } = require("@anders-t/ethers-ledger");
+
+async function main() {
```

Comment:
This script is for testing purposes only to verify that the agent mech creation works without issues.